adapters/smaato/smaatotest/video/multiple-adpods.json

```diff
@@ -0,0 +1,539 @@
+                }
+            },
+            {
+                "id": "2_2",
+                "video": {
+                    "mimes": [
+                        "video/mp4",
+                        "video/3gpp"
+                    ],
+                    "minduration": 5,
+                    "maxduration": 30,
+                    "protocols": [
+                        7
+                    ],
+                    "w": 1024,
+                    "h": 768,
+                    "startdelay": 0,
+                    "linearity": 1,
+                    "skip": 1,
+                    "skipmin": 5,
+                    "api": [
+                        7
+                    ]
+                },
+                "ext": {
+                    "bidder": {
+                        "publisherId": "1100042525",
+                        "adbreakId": "400000001"
+                    }
+                }
+            }
+        ],
+        "site": {
+            "publisher": {
+                "id": "1100042525"
+            },
+            "content": {
+                "title": "a-title",
+                "season": "a-season",
+                "series": "a-series",
+                "episode": 1,
+                "len": 900,
+                "livestream": 1
+            }
+        },
+        "device": {
+            "ua": "test-user-agent"
+        },
+        "user": {
+            "ext": {
+                "data": {}
+            }
+        }
+    },
+    "httpCalls": [
+        {
+            "expectedRequest": {
+                "headers": {
+                    "Content-Type": [
+                        "application/json;charset=utf-8"
+                    ],
+                    "Accept": [
+                        "application/json"
+                    ]
+                },
+                "uri": "https://prebid/bidder",
+                "body": {
+                    "id": "447a0a1d-389d-4730-a418-3777e95de7bd",
+                    "imp": [
+                        {
+                            "id": "1_1",
+                            "tagid": "400000001",
+                            "video": {
+                                "w": 1024,
+                                "h": 768,
+                                "mimes": [
+                                    "video/mp4",
+                                    "video/3gpp"
+                                ],
+                                "minduration": 5,
+                                "startdelay": 0,
+                                "linearity": 1,
+                                "maxduration": 30,
+                                "skip": 1,
+                                "protocols": [
+                                    7
+                                ],
+                                "skipmin": 5,
+                                "api": [
+                                    7
+                                ],
+                                "sequence": 1
```

Comment:
aha it starts with 1 ! I should change on js side, I hv it starting from 0 there